### PR TITLE
kie-server-tests: Avoid NPE when closing JAX-RS response

### PR DESCRIPTION
**Pull request by sutaakar**
To reveal real issue thrown when sending request.

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/test/java/org/kie/server/integrationtests/common/rest/RestMalformedRequestIntegrationTest.java [hunk 1]
```diff
@@ -61,7 +63,9 @@ public void testCreateContainerEmptyBody() throws Exception {
         } catch (Exception e) {
             throw new RuntimeException("Unexpected exception on empty body", e);
         } finally {
-            response.close();
+            if(response != null) {
```
Comment (triceo) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/test/java/org/kie/server/integrationtests/common/rest/RestMalformedRequestIntegrationTest.java]: My bad, thought it did.

**Comment by sutaakar** [comment]
jenkins retest this

**Comment by sutaakar** [comment]
jenkins retest this

**Comment by sutaakar** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by sutaakar** [comment]
jenkins retest this

**Pull request merged**